.github/workflows/appimage.yml

```diff
@@ -48,31 +46,20 @@ jobs:
     - name: make clean
       run: |
         chown -R testbot:testbot .
-        sudo -u testbot bash -lc 'cd viam-cartographer && make clean'
+        sudo -u testbot bash -lc 'make clean'
         
-    - name: make bufinstall buf setup
-      uses: nick-fields/retry@v2
-      with:
-        timeout_minutes: 5
-        max_attempts: 3
-        retry_on: timeout
-        shell: bash
-        command: |
-          chown -R testbot:testbot .
-          sudo -u testbot bash -lc 'cd viam-cartographer && make bufinstall buf setup'
-
-    - name: make build
+    - name: make setup
       run: |
-        sudo -u testbot bash -lc 'cd viam-cartographer && make build'
+        sudo -u testbot bash -lc 'make setup'
 
-    - name: make install-lua-files
+    - name: make build
       run: |
-        sudo -u testbot bash -lc 'cd viam-cartographer && make install-lua-files'
```

Comment:
[q] Why isn't `install-lua-files` needed anymore? Was it never needed during Appimage builds?